src/atwho.js

```diff
@@ -131,7 +131,16 @@ define([
                                                        options.property);
                             }
                             return value;
-                        }
+                        },
+                        afterMatchFailed: function(at, $el) {
+                            if (options.useRichText) {
+                                var content = $el.html().trim();
+                                if (richText.CASE_REF_REGEX.test(content) || form.isValidHashtag(content)) {
```

Comment:
thanks, fixed. but found that typing `#caze #case/thing` replaces the entire string with a bubble, instead of leaving `#caze` alone, so, not yet ready to merge.